2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -92,6 +93,23 @@ collector {
       }
     }
 
+
+    # config for NSQ sink
+    NSQ {
+      #Topic name for good raw event sink
+      good-sink: "{{collectorNsqSourceGoodTopicName}}"
+
+      #Topic name for bad event sink
+      bad-sink: "{{collectorNsqSinkTopicBadName}}"
+        
+      #Host name for NSQ tools
+      nsq-host: "{{NsqHost}}"
+
+      #Port for nsqd
+      nsqd-port: "{{NsqdPort}}"
```

Comment:
Actually there are two different tools for NSQ, one of them is nsqd and other is nsqlookupd. These two tools should run on the same machine therefore I used one host name which is nsq-host for both of them. Since they are two different tool, both of them have different port number.